src/Footer.tsx

```diff
@@ -0,0 +1,134 @@
+// need to convert to typescript
+
+import React from "react";
+// import {
+//   Box,
+//   Container,
+//   Row,
+//   Column,
+//   FooterLink,
+//   Heading,
+// } from "./FooterStyles"
+import {
+  Grid,
+  Link,
+  Stack,
+  ThemeProvider,
+  Typography,
+  createTheme,
+} from "@mui/material";
+document.body.style.color = "black";
+
+const Footer = () => {
+  let theme = createTheme({
+    components: {
+      MuiLink: {
+        styleOverrides: {
+          root: {
+            textDecoration: "none",
+            color: "black",
+            fontSize: "11px",
+            fontWeight: "600",
+            "&:hover": {
+              color: "#1db272",
+              transition: "200ms ease-in",
+            },
+          },
+        },
+      },
+    },
+    typography: {
+      h1: {
+        fontWeight: "600",
+        fontSize: "20px",
+        lineHeight: "20px",
+      },
+      h2: {
+        textDecoration: "underline",
+        fontSize: "13px",
+        fontWeight: "600",
+      },
+      h3: {
+        fontSize: "13px",
+        fontWeight: "400",
+      },
+    },
+  });
+
+  return (
+    <ThemeProvider theme={theme}>
+      {/*footer*/}
```

Comment:
Is an empty `{ }` required here? idk much about react

Reply:
i was just adding some comments to document the sections

Reply:
According to this, html comments are considered as DOM nodes in JSX so we need to wrap comments in curly braces. 
https://wesbos.com/react-jsx-comments

Reply:
😢